webui/fasttable.js

```diff
@@ -17,8 +17,8 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * $Revision$
- * $Date$
+ * $Revision: 778 $
+ * $Date: 2013-08-07 16:09:43 -0400 (Wed, 07 Aug 2013) $
```

Comment:
Please revert these two lines. Then I can merge the pull request without editing.

Looks like you were using the files from svn as the basis for the editing. The files from git doesn't have the revision and date as git doesn't support keywords (unfortunately).